mad_spark_multiagent/OUTPUT_FORMATS.md

```
@@ -0,0 +1,77 @@
+# MadSpark Output Formats Guide
+
+## Available Output Formats
+
+### 1. Summary Format (Default)
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)